libs/form-elements/src/FieldGroup.tsx

```diff
@@ -90,7 +90,7 @@ function FieldGroup({
   const fieldGroupTokens = useTokens("FieldGroup", props.fieldGroupTokens);
   const inputTokens = useTokens("Input", props.inputTokens);
 
-  const requiredLabelText = useLabel("required");
+  const requiredLabelText = useLabel("required", "Required field");
```

Comment:
can we use this as something like `useLabel("required", requiredLabelProp ?? "Required field")` so the user can override it?

Maybe even like `useLabel(requiredLabelProp, "required", "Required field")` so that even in the case of i18n the user can change the specific label on the spot?

Reply:
We agreed that this would be implemented additionally if needed